src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/misc/GMTE_AmazonPackager.java

```diff
@@ -127,134 +112,10 @@ public GT_Recipe.GT_Recipe_Map getRecipeMap() {
-                            || ItemList.Schematic_2by2.isStackEqual((Object) tInputItem)
-                            || ItemList.Schematic_3by3.isStackEqual((Object) tInputItem)) {
-                        mSchematics.put(new ItemStackData(tInputItem));
-                    }
-                }
-            }
-            if (mSchematics.size() > 0) {
-                for (ItemStackData g : mSchematics) {
-                    for (ItemStack tInputItem : tItems) {
-                        if (tInputItem != null) {
-                            mCompleted.put(new ItemStackData(tInputItem));
-                            checkRecipe(tInputItem, g.getStack());
-                        }
-                    }
-                }
-            }
-
-            return mCompleted != null && mCompleted.size() > 0;
-        }
-    }
-
-    public boolean checkRecipe(ItemStack inputStack, ItemStack schematicStack) {
-        if (GT_Utility.isStackValid((Object) inputStack) && GT_Utility.isStackValid((Object) schematicStack)
-                && GT_Utility.getContainerItem(inputStack, true) == null) {
-            ItemStack tOutputStack;
-            if (ItemList.Schematic_1by1.isStackEqual((Object) schematicStack) && inputStack.stackSize >= 1) {
-                tOutputStack = GT_ModHandler.getRecipeOutput(new ItemStack[] { inputStack });
-                if (tOutputStack != null && this.allowPutStack(tOutputStack, schematicStack)) {
-                    final ItemStack input = inputStack;
-                    --input.stackSize;
-                    this.lEUt = 32 * (1L << this.mTier - 1) * (1L << this.mTier - 1);
-                    // this.mMaxProgresstime = 16 / (1 << this.mTier - 1);
-                    this.mMaxProgresstime = 2;
-                    this.addOutput(tOutputStack);
-                    updateSlots();
-                    return true;
-                }
-                return false;
-            } else if (ItemList.Schematic_2by2.isStackEqual((Object) schematicStack) && inputStack.stackSize >= 4) {
-                tOutputStack = GT_ModHandler
-                        .getRecipeOutput(new ItemStack[] { inputStack, inputStack, null, inputStack, inputStack });
-                if (tOutputStack != null && this.allowPutStack(tOutputStack, schematicStack)) {
-                    final ItemStack input2 = inputStack;
-                    input2.stackSize -= 4;
-                    this.lEUt = 32 * (1L << this.mTier - 1) * (1L << this.mTier - 1);
-                    // this.mMaxProgresstime = 32 / (1 << this.mTier - 1);
-                    this.mMaxProgresstime = 4;
-                    this.addOutput(tOutputStack);
-                    updateSlots();
-                    return true;
-                }
-                return false;
-            } else if (ItemList.Schematic_3by3.isStackEqual((Object) schematicStack) && inputStack.stackSize >= 9) {
-                tOutputStack = GT_ModHandler.getRecipeOutput(
```

Comment:
This removes crafting recipe feature. I don't think many people are using this, but is it possible to keep it or do you give up?

Reply:
I admit that I don't know how many recipes are affected by it, but current amazon is very laggy. So if we don't loose too much, removing seems like the better option or?

Reply:
Seems ok. We don't have nugget / ingot manual compression recipes and dust recipes are covered by packager recipemap